### PR TITLE
TCM-776 | Dummy Permissions Alert

### PR DESCRIPTION
**Pull request by vespertineSiren**
### Summary

- Create PermissionAlert with viewstate and test utility

### Issue

[TCM-776](https://hyfnla.atlassian.net/browse/TCM-776)

### Video/Screenshot

Screen Shot 2021-07-27 at 6 08 30 PM
Screen Shot 2021-07-27 at 6 11 42 PM

### CODE REVIEW

File: app/src/main/java/com/trx/consumer/models/states/PermissionAlertViewState.kt [hunk 1]
```diff
@@ -0,0 +1,17 @@
+package com.trx.consumer.models.states
+
+import com.trx.consumer.R
+import com.trx.consumer.extensions.capitalized
+import com.trx.consumer.extensions.lowerCased
+
+enum class PermissionAlertViewState {
```
Comment (VivekV95) [inline app/src/main/java/com/trx/consumer/models/states/PermissionAlertViewState.kt:7]: There's an existing `PermissionsAlertViewState` in `develop` that can be used

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -135,6 +136,12 @@
     <string name="memberships_terms">Terms of Service</string>
     <string name="memberships_privacy">Privacy Policy</string>
 
+    <string name="permission_alert_title">%s Access Required</string>
+    <string name="permission_alert_camera_description">You need to give access to your camera to be able to join a live or virtual session</string>
+    <string name="permission_alert_microphone_description">You need to give access to your microphone to be able to join a live or virtual session</string>
+    <string name="permission_alert_primary_button">Turn On</string>
+    <string name="permission_alert_secondary_button">No, I will not join at this time</string>
```
Comment (VivekV95) [inline app/src/main/res/values/strings.xml:143]: From [slack](https://commafwd.slack.com/archives/G01JVMJ6G5S/p1627500530014200)

**Review by VivekV95 — CHANGES_REQUESTED** [review]
Couple comments and merge conflicts

File: app/src/main/res/values/strings.xml (hunk 2 above)
Comment (VivekV95) [reply app/src/main/res/values/strings.xml:143]: Nvm, label seems fine and makes sense

File: app/src/main/java/com/trx/consumer/models/states/PermissionAlertViewState.kt (hunk 1 above)
Comment (vespertineSiren) [reply app/src/main/java/com/trx/consumer/models/states/PermissionAlertViewState.kt:7]: Oops. Missed that. Will move over.

**Review by VivekV95 — APPROVED** [review]

**Review by onodude — APPROVED** [review]
thanks!

**Pull request merged**